### ISSUE TITLE
migration error in 1.0.0, works in 0.9.12

### ISSUE THREAD

**Issue by pusateri** [stefanfoulis/django-filer#603]
I ran 'python manage.py migrate' and got the following error after 'pip install django-filer'

Django (1.8.6)
django-mptt (0.7.4)
django-polymorphic (0.7.2)
easy-thumbnails (2.2)
Pillow (3.0.0)

``` python
Traceback (most recent call last):
  File "manage.py", line 10, in <module>
    execute_from_command_line(sys.argv)
  File "/usr/home/pusateri/.virtualenvs/prost/lib/python2.7/site-packages/django/core/management/__init__.py", line 354, in execute_from_command_line
    utility.execute()
  File "/usr/home/pusateri/.virtualenvs/prost/lib/python2.7/site-packages/django/core/management/__init__.py", line 346, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/usr/home/pusateri/.virtualenvs/prost/lib/python2.7/site-packages/django/core/management/base.py", line 394, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/usr/home/pusateri/.virtualenvs/prost/lib/python2.7/site-packages/django/core/management/base.py", line 445, in execute
    output = self.handle(*args, **options)
  File "/usr/home/pusateri/.virtualenvs/prost/lib/python2.7/site-packages/django/core/management/commands/migrate.py", line 93, in handle
    executor = MigrationExecutor(connection, self.migration_progress_callback)
  File "/usr/home/pusateri/.virtualenvs/prost/lib/python2.7/site-packages/django/db/migrations/executor.py", line 19, in __init__
    self.loader = MigrationLoader(self.connection)
  File "/usr/home/pusateri/.virtualenvs/prost/lib/python2.7/site-packages/django/db/migrations/loader.py", line 47, in __init__
    self.build_graph()
  File "/usr/home/pusateri/.virtualenvs/prost/lib/python2.7/site-packages/django/db/migrations/loader.py", line 182, in build_graph
    self.load_disk()
  File "/usr/home/pusateri/.virtualenvs/prost/lib/python2.7/site-packages/django/db/migrations/loader.py", line 123, in load_disk
    "creating Django migrations." % app_config.label
django.db.migrations.loader.BadMigrationError: Migrated app 'filer' contains South migrations. Make sure all numbered South migrations are deleted prior to creating Django migrations.
```

**Comment by jsma** [comment]
@mkoistinen there may be some issue with the package on PyPI. If I install from there, I get the error reported above. If I do `pip install -e git+https://github.com/stefanfoulis/django-filer.git@#egg=django-filer` or `pip install https://github.com/stefanfoulis/django-filer/archive/1.0.0.zip` it works.

**Comment by mattbriancon** [comment]
Seeing this as well. +1 on workaround from @jsma.

**Comment by mattbriancon** [comment]
Duplicate of #602.

**Comment by mkoistinen** [comment]
Fixed in v1.0.1

**Comment by pusateri** [comment]
Yes, just verified this fixed it. Thanks!
